content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -187,7 +187,7 @@ For a reference to old feature gates that are removed, please refer to
 | `SELinuxMountReadWriteOncePod` | `true` | Beta | 1.28 | |
 | `SchedulerQueueingHints` | `false` | Alpha | 1.28 | |
 | `SecurityContextDeny` | `false` | Alpha | 1.27 | |
-| `SidecarContainers` | `false` | Alpha | 1.28 | |
```

Comment:
Don't delete this line (history record), instead revise it to
```
| `SidecarContainers` | `false` | Alpha | 1.28 | 1.28 |
```

Reply:
Sorry I didn't check how this table was working, fixed!